### PR TITLE
tme: New package for The Machine Emulator 0.12rc9

### PR DESCRIPTION
**Pull request by phabrics**
The Machine Emulator, or tme, is a program that provides a general- purpose framework for computer emulation.  The first machine that tme could emulate was the Sun 2/120, one of the first widely available Sun workstations. Other emulations include the Sun 3/150, SPARCstation 2, and UltraSPARC 1.

### CODE REVIEW

File: mingw-w64-tme/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	 "${MINGW_PACKAGE_PREFIX}-libltdl")
+source=("https://osdn.net/users/phabrics/pf/${_realname}/dl/${_realname}-${pkgver}.tar.xz")
+sha256sums=('28499e869f09e1f3c8ca76ed97f29bd647db3cd015e09c660dfe30193bed7763')
+
+build()
+{
+    cd "${srcdir}/${_realname}-${pkgver}"
+    mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
+
+    ../"${_realname}-${pkgver}"/configure \
```
Comment (MehdiChinoune) [inline mingw-w64-tme/PKGBUILD:28]:
```suggestion
    mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"

    if [[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]]; then
        CFLAGS+=" -Wno-int-conversion"
    fi

    ../"${_realname}-${pkgver}"/configure \
```

File: mingw-w64-tme/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	     "texinfo")
+depends=("${MINGW_PACKAGE_PREFIX}-gtk3"
+	 "${MINGW_PACKAGE_PREFIX}-libvncserver"
+	 "${MINGW_PACKAGE_PREFIX}-libltdl")
+source=("https://osdn.net/users/phabrics/pf/${_realname}/dl/${_realname}-${pkgver}.tar.xz")
+sha256sums=('28499e869f09e1f3c8ca76ed97f29bd647db3cd015e09c660dfe30193bed7763')
+
+build()
+{
+    cd "${srcdir}/${_realname}-${pkgver}"
```
Comment (MehdiChinoune) [inline mingw-w64-tme/PKGBUILD:25]:
```suggestion
```
not needed

File: mingw-w64-tme/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Maintainer: Ruben Agin <phabrics@phabrics.com>
+_realname=tme
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.12rc9
+pkgrel=1
+pkgdesc="The Machine Emulator, or ${_realname}, provides a general-purpose framework for computer emulation."
```
Comment (MehdiChinoune) [inline mingw-w64-tme/PKGBUILD:7]:
```suggestion
pkgdesc="The Machine Emulator, or ${_realname}, provides a general-purpose framework for computer emulation. (mingw-w64)"
```

File: mingw-w64-tme/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Maintainer: Ruben Agin <phabrics@phabrics.com>
+_realname=tme
```
Comment (MehdiChinoune) [inline mingw-w64-tme/PKGBUILD:2]:
```suggestion
# Maintainer: Ruben Agin <phabrics@phabrics.com>

_realname=tme
```

File: mingw-w64-tme/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+license=('GPL3')
+url="https://osdn.net/users/phabrics/pf/${_realname}/wiki/FrontPage"
+makedepends=("${MINGW_PACKAGE_PREFIX}-autotools"
+             "${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-pkg-config"
+	     "bison"
+	     "texinfo")
+depends=("${MINGW_PACKAGE_PREFIX}-gtk3"
+	 "${MINGW_PACKAGE_PREFIX}-libvncserver"
+	 "${MINGW_PACKAGE_PREFIX}-libltdl")
```
Comment (MehdiChinoune) [inline mingw-w64-tme/PKGBUILD:19]:
```suggestion
	     "${MINGW_PACKAGE_PREFIX}-libvncserver"
	     "${MINGW_PACKAGE_PREFIX}-libltdl")
```

File: mingw-w64-tme/PKGBUILD (hunk 4 above)
Comment (phabrics) [reply mingw-w64-tme/PKGBUILD:2]: OK

File: mingw-w64-tme/PKGBUILD (hunk 3 above)
Comment (phabrics) [reply mingw-w64-tme/PKGBUILD:7]: Sounds good.

File: mingw-w64-tme/PKGBUILD (hunk 2 above)
Comment (phabrics) [reply mingw-w64-tme/PKGBUILD:25]: I didn't know that. I will commit these suggested changes. Is that all I have to do? 

Thanks for your help!

**Pull request merged**